docs/content/documentation/systems/Dampers/tensor_mechanics/ElementJacobianDamper.md

```diff
@@ -0,0 +1,11 @@
+# ElementJacobianDamper
+!description /Dampers/ElementJacobianDamper
+
+This damper limits the change in the Jacobians of elements. The damper becomes active if the relative change in Jacobian of any element exceeds the user defined maximum.
+
+!parameters /Dampers/ElementJacobianDamper
+
+!inputfiles /Dampers/ElementJacobianDamper
+moose/modules/tensor_mechanics/tests/jacobian_damper/cube_load.i
```

Comment:
I think the system is supposed to automatically generate links to all of the tests that use it or something.  I tried to look at the page to see what it looks like, but it looks like Dampers don't even get included in the documentation currently.  I guess I'd say you should remove it.